chore: Update bug report template (#889)
newrelic/newrelic-browser-agent: .github/ISSUE_TEMPLATE/bug_report.md

## Relevant Logs / Console output

> [NOTE]: # ( Please provide specifics of the local error logs, Browser Dev Tools console, etc. if appropriate and possible. Include all--if possible--or any of the following:)

Reviewer: ```suggestion
[NOTE]: # ( Please provide specifics of the local error logs, Browser Dev Tools console, etc. Include as many of the following as possible:)
```

> * screenshots showing errors
> * page source from the page in question

Reviewer: ```suggestion
* Screenshots showing errors
* Page source from the page in question
```
